api/v1alpha1/dspipeline_types.go

```diff
@@ -133,6 +133,10 @@ type APIServer struct {
 	// +kubebuilder:default:=true
 	// +kubebuilder:validation:Optional
 	AutoUpdatePipelineDefaultVersion bool `json:"autoUpdatePipelineDefaultVersion"`
+	// This is the path where the ca bundle will be mounted in the
+	// pipeline server and user executor pods
+	// +kubebuilder:validation:Optional
+	CABundleFileMountPath string `json:"caBundleFileMountPath"`
```

Comment:
@VaniHaripriya can you update this file also with this new field: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/samples/v2/dspa-all-fields/dspa_all_fields.yaml 

this file is just an example to list all available crd fields, so you can put any example like: 

```yaml
caBundleFileMountPath:  /your/certbundle/path.crt
```